components/user_settings/sidebar/user_settings_sidebar.jsx

```diff
@@ -290,10 +314,99 @@ export default class UserSettingsSidebar extends React.Component {
         );
     };
 
+    renderChannelSwitcherSection = () => {
+        let channelSwitcherSectionDescId = 'user.settings.sidebar.channelSwitcherSectionDesc.windows';
+        let channelSwitcherSectionDescDefault = 'The channel switcher is shown at the bottom of the sidebar and is used to jump between channels quickly. It can also be accessed using CTRL + K.';
+        if (isMac()) {
+            channelSwitcherSectionDescId = 'user.settings.sidebar.channelSwitcherSectionDesc.mac';
```

Comment:
When you rebase from master, be sure this string is wrapped inside the `t` function from `utils/i18n`.

Reply:
done